client/src/pages/Login.jsx

```
@@ -1,5 +1,37 @@
+
   return (
-    <div>Login</div>
-  )
+    <div className={styles.wrapper}>
+      <img src="logo.png" alt="muse" style={{ width: '10%' }} className={styles.logo} />
+      <p className={styles.description}>책의 순간을 한 줄로 기록하는 공간</p>
+      <button
+        className={styles.loginButton}
+        onClick={() => (window.location.href = KAKAO_AUTH_URL)}
+      >
+        <img src={kakao} alt="kakaoLogin" className={styles.loginButtonImage} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)